recipes/antlr4-cppruntime/all/patches/0004-update-DropLibuuid-4.10.x-1.patch

```diff
@@ -0,0 +1,13 @@
+--- runtime/Cpp/CMakeLists.txt
++++ runtime/Cpp/CMakeLists.txt
+@@ -41,10 +41,6 @@
+   CMAKE_POLICY(SET CMP0054 OLD)
+ endif()
+ 
+-if(CMAKE_SYSTEM_NAME MATCHES "Linux")
+-  find_package(PkgConfig REQUIRED)
+-  pkg_check_modules(UUID REQUIRED uuid)
```

Comment:
How can this be removed without changing the code? 😕

Reply:
This PR [[C++] Remove libuuid dependency](https://github.com/antlr/antlr4/pull/3787) showed that `libuuid` is not used since at least ANTLR 4.9. You can find the conclusion at here: https://github.com/antlr/antlr4/pull/3787#issuecomment-1206106174.